projects/cds/src/merchandising/adapters/strategy/cds-merchandising-strategy.adapter.ts

```diff
@@ -1,11 +1,9 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { ConverterService } from '@spartacus/core';
 import { Observable } from 'rxjs';
 import { CdsEndpointsService } from '../../../services/cds-endpoints.service';
-import { MERCHANDISING_PRODUCTS_NORMALIZER } from '../../connectors/strategy/converters';
 import { MerchandisingStrategyAdapter } from '../../connectors/strategy/merchandising-strategy.adapter';
-import { MerchandisingProducts } from '../../model/merchandising-products.model';
+import { StrategyProducts } from '../../model';
```

Comment:
don't forgot to specifically say index.ts 

this may cause problems building for production and would be hard to catch later on

Reply:
Or as you did above, state the specific file

import { StrategyProducts } from '../../model/strategy-products.model';

Reply:
Corrected